docs/versioning.rst

```diff
@@ -7,10 +7,15 @@ The functionality is optional and has to be explicitly enabled, note that:
 
 * SCM tags are used for versioning
 
-* only git is supported currently (contributions for other SCMs welcome and should be easy, see `Scm class`_)
+# XXX: s/git/Git (but also throughout then) ?
+* only git is supported currently (contributions for other SCMs are welcome and should be easy, see `Scm class`_)
 
 * It can be a drop-in replacement for setuptools_scm_
 
+# XXX: also "dev" and "devcommit"?!
+# XXX: the code also handles "changes" (https://github.com/zsimic/setupmeta/blob/586102493f7167d47d633ff46eb9945ae36b730c/setupmeta/versioning.py#L305)
+# XXX: ...and "default", and "tag"
+# XXX: s/strategies/formats/ ?  (otherwise vice-versa below in the heading)
```

Comment:
Ah yes, looks like I missed `default` and `tag` (which are the same as `post`)... should maybe remove those 2?